staging/src/k8s.io/apimachinery/go.mod

```diff
@@ -46,6 +46,7 @@ require (
 	github.com/onsi/gomega v1.27.4 // indirect
 	github.com/pkg/errors v0.9.1 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/rogpeppe/go-internal v1.10.0 // indirect
```

Comment:
This change add go-internal to apimachinery, code-generator, etc and removed it from api, kube-proxy, etc.

Some more context:

```sh
# for apimachinery
$ go mod why -m github.com/rogpeppe/go-internal
# github.com/rogpeppe/go-internal
k8s.io/apimachinery/pkg/runtime
sigs.k8s.io/structured-merge-diff/v4/value
gopkg.in/yaml.v2
gopkg.in/yaml.v2.test
gopkg.in/check.v1
github.com/kr/pretty
github.com/rogpeppe/go-internal/fmtsort

# for api
$ go mod why -m github.com/rogpeppe/go-internal
# github.com/rogpeppe/go-internal
k8s.io/api/admission/v1
k8s.io/apimachinery/pkg/runtime
sigs.k8s.io/structured-merge-diff/v4/value
gopkg.in/yaml.v2
gopkg.in/yaml.v2.test
gopkg.in/check.v1
github.com/kr/pretty
github.com/rogpeppe/go-internal/fmtsort
````

Reply:
I need to set aside more time to reason through this... this is surprising to me... I would have expected `// indirect` entries to already exist for all the modules which had paths to this dependency, which included apimachinery before this PR.